cmd/soroban-cli/src/commands/network/container/start.rs

```diff
@@ -82,17 +82,31 @@ impl Runner {
         let docker = self.args.container_args.connect_to_docker().await?;
 
         let image = self.get_image_name();
-        docker
-            .create_image(
-                Some(CreateImageOptions {
-                    from_image: image.clone(),
-                    ..Default::default()
-                }),
-                None,
-                None,
-            )
-            .try_collect::<Vec<_>>()
-            .await?;
+        let mut stream = docker.create_image(
+            Some(CreateImageOptions {
+                from_image: image.clone(),
+                ..Default::default()
+            }),
+            None,
+            None,
+        );
+
+        while let Some(result) = stream.try_next().await.transpose() {
+            if let Ok(item) = result {
+                if let Some(status) = item.status {
+                    if status.contains("Pulling from")
+                        || status.contains("Digest")
+                        || status.contains("Status")
+                    {
+                        self.print.infoln(status);
+                    }
+                }
+            } else {
+                self.print.warnln("Failed to fetch image from Docker Hub.");
+                self.print.warnln("Attempting to start local quickstart image instead. Please note this image may be out-of-date.");
```

Comment:
Suggested abbreviated messaging:
```suggestion
                self.print.warnln("Attempting to start local quickstart image. The image may be out-of-date.");
```